crates/holochain_cascade/src/authority/test.rs

```diff
@@ -295,3 +294,108 @@ async fn get_agent_activity() {
 
     pretty_assertions::assert_eq!(result, expected);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn get_agent_activity_respects_withhold_publish() {
+    holochain_trace::test_run();
+    let db = test_dht_db();
```

Comment:
Should this be an authored db? Not that it currently matters, but semantically it's true, right?